1:30 - Update Events (#890)
FusionAuth/fusionauth-site: site/docs/v1/tech/events-webhooks/event-bodies/user-registration-update-complete.adoc

:type: user.registration.update.complete
== User Registration Update Complete

[NOTE.since]
====
This event has been available since 1.30.0
====

> This event is generated when a user registration update transaction has completed.

Reviewer: Maybe omit transaction here and describe it below in the Transaction Compatible section?

[source,shell]
.Event type
[subs="attributes"]
----
{type}
----

=== Event Scope

include::../_tenant-or-application-scoped-event.adoc[]

> 

Reviewer: Add `Transaction Compatible` section.

=== Event Body

[.api]
[field]#event.applicationId# [type]#[UUID]#::
The unique Id of the Application for which the user has now been registered.

[field]#event.createInstant# [type]#[Long]#::
The link:/docs/v1/tech/reference/data-types/#instants[instant] that the event was generated.

[field]#event.id# [type]#[UUID]#::
The unique Id of the event. You may receive an event more than once based upon your transaction settings. This Id may be used to identify a duplicate event.

include::_event-info.adoc[]

[field]#event.original# [type]#[Object]#::
The original registration prior to being updated. See the link:/docs/v1/tech/apis/registrations/[Registration API] for property definitions and example JSON.

[field]#event.registration# [type]#[Object]#::

> The user registration that has been updated. See the link:/docs/v1/tech/apis/registrations/[Registration API] for property definitions and example JSON.

Reviewer: ```suggestion
The user registration with current, updated values. See the link:/docs/v1/tech/apis/registrations/[Registration API] for property definitions and example JSON.
```

[field]#event.tenantId# [type]#[UUID]#::

> The unique tenant identifier. This value may not be returned if not applicable.

Reviewer: when would this not be applicable?
